src/etos_lib/opentelemetry/semconv.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Axis Communications AB.
+#
+# For a full list of individual contributors, please see the commit history.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ETOS OpenTelemetry semantic conventions module."""
+
+class Attributes:
+    """Constants for ETOS OpenTelemetry semantic conventions."""
+
+    # General ETOS conventions
+    SUITE_ID = "etos.suite.id"
+    SUBSUITE_ID = "etos.subsuite.id"
+
+    # Testrunner conentions
+    TESTRUN_ID = "etos.testrun.id"
+    TESTRUNNER_ID = "etos.testrunner.id"
+    TESTRUNNER_EXCEPTION = "etos.testrunner.exception"
```

Comment:
This shouldn't be necessary? Exception information should be added automatically when we use python and it will be irrelevant if we update to use Golang instead

Reply:
In  Environment Provider it exceptions are  sometimes returned as variables, and they aren't added automatically.

        success, exception = checkin_provider(Iut(**iut), IutProvider(etos, jsontas, iut_ruleset))
        if not success:
            failure = exception

In cases like this, they seem more like application-layer exceptions.

Reply:
You just added record_exception in the environment provider review, making this attribute unnecessary?

Reply:
Replaced set_attribute() with record_exception().